shells/mrsh/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           sourcehut 1.0
+
+sourcehut.setup     emersion mrsh a858396b
+version             0.0.0
+revision            0
+platforms           any
+checksums           rmd160  ba54f6093edf4fa1f8a60fee3e8f469ac26d134b \
+                    sha256  d7fa79295a51ae30f4476499b2c0506223a4e18b6a5b696c00ece0866c3c6861 \
+                    size    83745
+
+description         A minimal POSIX shell.
+long_description \
+    mrsh is a minimal shell striving for POSIX compliance, no less, no more.
+
+categories          shells
+maintainers         nomaintainer
+license             MIT
+
+depends_build       path:bin/pkg-config:pkgconfig
+
+patchfiles-append   configure.patch \
+                    frontend-readline.c.patch
+
+compiler.c_standard 1999
```

Comment:
I'm not sure what the default is. It probably is C99, but the configure script explicitly checks for `-std=c99` so I thought it may make sense to also specify it in the Portfile for an appropriate compiler selection.

Defaults can change over time.

Are there any disadvantages when setting `compiler.c_standard`?

Reply:
Yah, please don't set it if it's not necessary.

Reply:
default is actually `1989` (https://trac.macports.org/wiki/CompilerSelection) so it might be required after all - that's why I told you to double check this...

Reply:
Yes, @reneeotten, the code seems to agree with the default being 1989 (cf. https://github.com/macports/macports-base/blob/v2.8.1/src/port1.0/portconfigure.tcl#L349).

In any case I still don't understand the reasoning why it is considered bad to specify `compiler.c_standard` if the project explicitly says they're targeting a specific C standard version.

Do we have a consensus? Should the line stay or be removed?

Reply:
If it's really the case that the default standard is the old ANSI stuff that should be fixed and set to C99. Regardless: I don't think this has any trouble building without that being set does it? Generally that flag doesn't do what you think it does. It tells MacPorts not to allow the use of a compiler that can't support the given standard. However, I don't think a single C compiler in MacPorts won't support C99.

Reply:
@reneeotten Yes, I didn't check, because I was not interested in the result and I thought my question was clear, but apparently it wasn't.

What I wanted to know is the reasoning why the `c_standard` should not be included in the Portfile when a software explicitly describes itself as being written in said C standard _even if_ it is the same as the MacPorts default.
The software will stay being written in C99 also if MacPorts changes the default C standard.

This question is still unanswered and completely unrelated to the current or future MacPorts default `compiler.c_standard` value.

Should I add the `c_standard` back now or not since @pmetzger voiced his intent to change the default to 1999?

Reply:
We don't add default settings that are already set by base or PortGroups - why would we make an exception for this setting? 

But as it is not the default here you should set it in this case.

Reply:
OK, I added it back.

Removing a line when defaults change is always easier than figuring out where new ones are needed :-)

Reply:
I don't get why we would have needed this but okay, it's harmless enough.